vnext/ReactCommon/TEMP_UntilReactCommonUpdate/react/renderer/mapbuffer/MapBuffer.cpp

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
Is this a straight copy?  Whats the diff here?  Or is it for #include paths?

Reply:
Forgot to mark it - I've added the static_cast on line 16. Fix should get merged in core soon, and we'll ingest it then.
This is the PR: https://github.com/facebook/react-native/pull/31106

Reply:
Could you add this as an override?

Reply:
And also add this directory to "includePatterns" at the top of overrides.json? That way the tool will prompt people to register them.

Reply:
Good call, will add now